lua/bufdelete/init.lua

```diff
@@ -40,7 +40,7 @@ local function buf_kill(range, force, wipeout)
 
     -- If force is disabled, check for modified buffers in range.
     if not force then
-        for bufnr, _ in pairs(target_buffers) do
+        for bufnr, buf in pairs(target_buffers) do
```

Comment:
why did you make this change?

Reply:
Sorry, that was an artifact from a different approach I was taking. I just forgot to remove it.

Reply:
Removed.